test/kafo_wizards/factory_test.rb

```diff
@@ -48,8 +48,11 @@
   end
 
   it "raises NameError for unknown type" do
-    e = Proc.new { factory.nonsense(:sth) }.must_raise NameError
-    e.message.must_equal "Unknown type of entry (nonsense)"
+    error = assert_raises(NameError) do
+      factory.nonsense(:sth)
+    end
+    expected_error_message = "Unknown type of entry (nonsense)"
+    assert_match expected_error_message, error.message
```

Comment:
I think this should work
```suggestion
    assert_equal expected_error_message, error.name
```

Reply:
This didn't work and making this changes creating another 2 failures.

```
  1) Failure:
KafoWizards::Entries::NetmaskEntry::validate#test_0004_raises ValidationError when the netmask in bitmask mode is invalid [/home/akumari/Desktop/installer/kafo_wizards/test/kafo_wizards/entries/netmask_test.rb:35]:
--- expected
+++ actual
@@ -1 +1 @@
-"127001/56 is not valid netmask (invalid length)"
+"127001/56 is not valid netmask (invalid length: 4294967295)"


  2) Failure:
KafoWizards::Factory#test_0009_raises NameError for unknown type [/home/akumari/Desktop/installer/kafo_wizards/test/kafo_wizards/factory_test.rb:55]:
--- expected
+++ actual
@@ -1 +1 @@
-"Unknown type of entry (nonsense)"
+nil
```
But I guess this is again another ruby compatibility issue, because the below change work son <3.1 only
https://github.com/theforeman/kafo_wizards/blob/6fede92c9315babdf2d17dd17ee2a7a117a09693/test/kafo_wizards/entries/netmask_test.rb#L34

and this change works on ruby >=3.1
https://github.com/theforeman/kafo_wizards/blob/a0fd0128143a6ea4b5e81d38303d6ae89883b476/test/kafo_wizards/entries/netmask_test.rb#L34

I was thinking if we can handle this based on the Ruby version through conditional statement is it a good way to do that? WDYT? or if we have another ways to satify this?